webpack/assets/javascripts/react_app/components/HostDetails/ExperimentalAlert.js

```diff
@@ -0,0 +1,55 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import {
+  Alert,
+  AlertActionCloseButton,
+  AlertActionLink,
+} from '@patternfly/react-core';
+import { translate as __ } from '../../common/I18n';
+import { visit } from '../../../foreman_navigation';
+import { foremanUrl } from '../../common/helpers';
+
+const ExperimentalAlert = ({ hostId }) => {
+  const [alertVisibility, setAlertVisibility] = useState(true);
+  if (!alertVisibility) return null;
+  return (
+    <Alert
+      style={{ margin: '15px 20px' }}
+      isInline
+      variant="info"
+      title={__(
+        'This page redesign is experimental and open for further evolution'
+      )}
+      actionClose={
+        <AlertActionCloseButton onClose={() => setAlertVisibility(false)} />
+      }
+      actionLinks={
+        <>
+          <AlertActionLink
+            onClick={() =>
+              visit(
```

Comment:
I think it would make sense to open it in a new tab so the user can see what they give feedback on

Reply:
> It should open a new tab. (I thought @amirfefer presented it that way on our sync)